layouts/partials/videos/card.html

```diff
@@ -10,7 +10,7 @@
             {{ .video.title }}
           </div>
           <div class="video-description">
-            {{ .video.description | markdownify }}
+            {{ (substr (.video.description | replaceRE "https?://([^ ]+)" "") 0 140) }}
```

Comment:
I *think* this would work, but FYI there is a `truncate` (https://gohugo.io/functions/truncate/) method that can ad elipsis 

```suggestion
            {{ .video.description | replaceRE "https?://([^ ]+)" "" | truncate 140 }}
```

Reply:
Nice thanks dude